examples/meta/src/converter/independent_component_analysis_fast.sg

```diff
@@ -19,3 +19,7 @@ Features converted = ica.transform(feats)
 RealMatrix mixing_matrix = ica.get_real_matrix("mixing_matrix")
 RealMatrix unmixed_signal = converted.get_real_matrix("feature_matrix")
 #![extract]
+
+#![inverse_transform]
+Features feats_original = ica.inverse_transform(converted)
```

Comment:
pls also extract the matrix here (even if it is not shown in the cookbook) for the sake of integration testing ...